File: HotDesk_PRAI/Backup/Library/Library.Database/Others/ListCollection.vb
Public Class ListCollection
    Private _TotalRow As Integer = 0
    Public Property TotalRow() As Integer
        Get
            Return _TotalRow
        End Get
        Set(ByVal value As Integer)
            _TotalRow = value
        End Set
    End Property

    Private _Data As DataTable = New DataTable
    Public Property Data() As DataTable
        Get
            Return _Data
        End Get
        Set(ByVal value As DataTable)
            _Data = value
        End Set
    End Property
End Class

File: HotDesk_PRAI/App_Code/listcollection.cs
using System.Data;

public class listcollection
{
    private int _TotalRow = 0;
    public int TotalRow
    {
        get { return _TotalRow; }
        set { _TotalRow = value; }
    }

    private DataTable _Data = new DataTable();
    public DataTable Data
    {
        get { return _Data; }
        set { _Data = value; }
    }
}
